docs/painless/painless-guide/painless-execute-script.asciidoc

```diff
@@ -77,8 +77,9 @@ sorted list of `double` values. See
 <<painless-runtime-double,double_field context>>.
 
 `geo_point_field`::
-The context for {ref}/geo-point.html[`geo-point` fields]. `emit` takes a
-`geo-point` value and the script returns coordinates for the geo point. See
+The context for {ref}/geo-point.html[`geo-point` fields]. `emit` takes two double
```

Comment:
@iverase Could you verify this change. I think the old description seemed quite incorrect.

Reply:
I had a look and yes the explanation of the emit is correct, thanks! What it is not clear to me what we refer here to what the script returns. This is currently a bug as it should behave as an indexed geo_point which is not the case.

Reply:
Ok, this geo_point context is different to the out put of the fields API. lThe change looks good to me then.